platforms/chibios/chibios_config.h

```diff
@@ -50,9 +50,15 @@
 #        define I2C1_CLOCK_SPEED 400000
 #    endif
 
-#    define SPI_SCK_PAL_MODE (PAL_MODE_ALTERNATE_SPI | PAL_RP_PAD_SLEWFAST | PAL_RP_PAD_DRIVE4)
-#    define SPI_MOSI_PAL_MODE SPI_SCK_PAL_MODE
-#    define SPI_MISO_PAL_MODE SPI_SCK_PAL_MODE
+#    ifndef SPI_SCK_PAL_MODE
+#        define SPI_SCK_PAL_MODE (PAL_MODE_ALTERNATE_SPI | PAL_RP_PAD_SLEWFAST | PAL_RP_PAD_DRIVE4)
+#    endif
+#    ifndef SPI_MOSI_PAL_MODE
+#        define SPI_MOSI_PAL_MODE SPI_SCK_PAL_MODE
+#    endif
+#    ifndef SPI_MISO_PAL_MODE
+#        define SPI_MISO_PAL_MODE SPI_SCK_PAL_MODE
+#    endif
```

Comment:
With this code redefining just `SPI_SCK_PAL_MODE` would result in `SPI_MOSI_PAL_MODE` and `SPI_MISO_PAL_MODE` also using the same values; this probably won't be expected.  Maybe we need to duplicate the value here to avoid this unexpected (and undocumented) interaction between settings.
```suggestion
#    ifndef SPI_MOSI_PAL_MODE
#        define SPI_MOSI_PAL_MODE (PAL_MODE_ALTERNATE_SPI | PAL_RP_PAD_SLEWFAST | PAL_RP_PAD_DRIVE4)
#    endif
#    ifndef SPI_MISO_PAL_MODE
#        define SPI_MISO_PAL_MODE (PAL_MODE_ALTERNATE_SPI | PAL_RP_PAD_SLEWFAST | PAL_RP_PAD_DRIVE4)
#    endif
```